continuous-integration/travis/regress.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh -x
+
+echo "Install Wit/Wake"
+
+git clone https://github.com/sifive/wit.git
+export PATH=$PATH:$PWD/wit
+
+export USE_FUSE_WAKE=0
+rm -r wake # temporary solution to duplicate naming issue
+git clone https://github.com/sifive/wake.git
+cd wake
+git checkout v0.15
```

Comment:
Can we just install the binary release of wake? https://github.com/sifive/wake/releases/tag/v0.15.1

Reply:
what about for mac os? should I move this into `install_deps.sh`?

Reply:
Yeah, this is installing a dependency, so that should really go into a different file.

Also, on a completely separate note, can we rename this from `regress.sh` to something else? To regress is to go back to an earlier state; regression testing is the act of testing to detect unintentionally regressions. "Regressing" is a bad thing, and this script should not be regressing, but rather checking for regressions. I think `test.sh` is fine enough.

Reply:
does using the release actually save us anything? maybe i'm just using it wrong. running
```
wget wake.deb
sudo dpkg -i wake.deb
```
just does the same thing of checking out `v0.15.0` then running `make`

Reply:
Using the release should obviate the need to compile Wake itself. It should be the same as installing any other Ubuntu binary package and allowing you to just execute the Wake binary without having to compile it from source.

Reply:
oh wait I'm dumb. I forgot to remove the checkout and make stuff from the test script.